example-schemas/swagger-petstore/generated/petstore/pet.bdl

```diff
@@ -0,0 +1,266 @@
+# standard - swagger-petstore
+
+import swagger.petstore.entity { ApiResponse, Pet, PetStatus }
+
+@ http - PUT /pet
+@ security - petstore_auth: write:pets, read:pets
+@ tags - pet
+@ summary - Update an existing pet.
+@ description - Update an existing pet by Id.
+proc UpdatePet = Pet -> UpdatePetOutput throws UpdatePetError
+
+oneof UpdatePetOutput {
+  @ status - 200
+  @ description - Successful operation
+  Pet,
+}
+
+oneof UpdatePetError {
+  @ status - 400
+  @ description - Invalid ID supplied
+  void,
+
+  @ status - 404
+  @ description - Pet not found
+  void,
+
+  @ status - 422
+  @ description - Validation exception
+  void,
+
+  @ status - default
+  @ description - Unexpected error
+  void,
+}
+
+@ http - POST /pet
+@ security - petstore_auth: write:pets, read:pets
+@ tags - pet
+@ summary - Add a new pet to the store.
+@ description - Add a new pet to the store.
+proc AddPet = Pet -> AddPetOutput throws AddPetError
+
+oneof AddPetOutput {
+  @ status - 200
+  @ description - Successful operation
+  Pet,
+}
+
+oneof AddPetError {
+  @ status - 400
+  @ description - Invalid input
+  void,
+
+  @ status - 422
+  @ description - Validation exception
+  void,
+
+  @ status - default
+  @ description - Unexpected error
+  void,
+}
+
+@ http - GET /pet/findByStatus
+@ security - petstore_auth: write:pets, read:pets
+@ tags - pet
+@ summary - Finds Pets by status.
+@ description - Multiple status values can be provided with comma separated strings.
+proc FindPetsByStatus =
+  FindPetsByStatusInput ->
+  FindPetsByStatusOutput throws FindPetsByStatusError
+
+struct FindPetsByStatusInput {
+  @ in - query
+  @ description - Status values that need to be considered for filter
+  status?: PetStatus,
```

Comment:
this `status` query is annotated as `explode: true` in openapi specification, but it is not an array type. should we have to treat this as an array of enum in bdl result?

Reply:
From the schema description, it looks like it should be an array type.
Let's add the `@ explode` attribute as well.